pandas/core/internals/managers.py

```diff
@@ -1515,25 +1515,14 @@ class SingleBlockManager(BlockManager):
     def __init__(
         self,
         block: Block,
-        axis: Union[Index, List[Index]],
+        axis: Index,
         do_integrity_check: bool = False,
         fastpath: bool = False,
```

Comment:
`fastpath` has no effect anymore, now? It is still being used in places internally?